doc/source/getting_started/comparison/includes/column_operations.rst

```diff
@@ -1,4 +1,4 @@
-pandas provides similar vectorized operations by specifying the individual ``Series`` in the
+pandas provides vectorized operations by specifying the individual ``Series`` in the
```

Comment:
This include is now used in the SAS, Stata, and spreadsheet pages. Since spreadsheets don't have "vectorized operations" in the same way, took out this reference.